text/0000-rust-governance.md

```diff
@@ -0,0 +1,728 @@
+most changes to the standard library, and
+[a few other topics](https://github.com/rust-lang/rfcs#when-you-need-to-follow-this-process).
+RFCs are either closed immediately (if they are clearly not viable),
+or else assigned a *shepherd* who is responsible for keeping the
+discussion moving and ensuring all concerns are responded to.
+
+The final decision to accept or reject an RFC is made by the core
+team. In many cases this decision follows after many rounds of
+consensus-building among all stakeholders for the RFC. In the end,
+though, most decisions are about weighting various tradeoffs, and the
+job of the core team is to make the final decision about such
+weightings in light of the overall direction of the language.
+
+## What needs improvement
+
+At a high level, we need to improve:
+
+* Process scalability.
+* Stakeholder involvement.
+* Clarity/transparency.
+* Moderation processes.
+
+Below, each of these bullets is expanded into a more detailed analysis
+of the problems. These are the problems this RFC is trying to
+solve. The "Detailed Design" section then gives the actual proposal.
+
+### Scalability: RFC process
+
+In some ways, the RFC process is a victim of its own success: as the
+volume and depth of RFCs has increased, it's harder for the entire
+core team to stay educated and involved in every RFC. The
+[shepherding process](https://github.com/rust-lang/rfcs#the-role-of-the-shepherd)
+has helped make sure that RFCs don't fall through the cracks, but even
+there it's been hard for the relatively small number of shepherds to
+keep up (on top of the other work that they do).
+
+Part of the problem, of course, is due to the current push toward 1.0,
+which has both increased RFC volume and takes up a great deal of
+attention from the core team. But after 1.0 is released, the community
+is likely to grow significantly, and feature requests will only
+increase.
+
+Growing the core team over time has helped, but there's a practical
+limit to the number of people who are jointly making decisions and
+setting direction.
+
+A distinct problem in the other direction has also emerged recently: we've
+slowly been requiring RFCs for increasingly minor changes. While it's important
+that user-facing changes and commitments be vetted, the process has started to
+feel heavyweight (especially for newcomers), so a recalibration may be in order.
+
+We need a way to scale up the RFC process that:
+
+* Ensures each RFC is thoroughly reviewed by several people with
+  interest and expertise in the area, but with different perspectives
+  and concerns.
+
+* Ensures each RFC continues moving through the pipeline at a
+  reasonable pace.
+
+* Ensures that accepted RFCs are well-aligned with the values, goals,
+  and direction of the project, and with other RFCs (past, present,
+  and future).
+
+* Ensures that simple, uncontentious changes can be made quickly, without undue
+  process burden.
```

Comment:
A *minor* nitpick: I would say "unnecessary" or "needless" rather than "undue". The phrase "undue process burden" sounds like a reference to the concept of "due process", which is well-known legal jargon in most English-speaking countries, but I think that's just a distracting coincidence.